### ISSUE TITLE
how to select hyperparameters?

### ISSUE THREAD

**Issue by zqk-7k** [benmoseley/harmonic-oscillator-pinn-workshop#2]
I noticed that in using Physics-Informed Neural Networks (PINNs) to address the current problem, you selected the following hyperparameters for the loss weights:
lambda1, lambda2 = 1e-1, 1e-4
After experimenting with modifying these hyperparameters, the simulation performance indeed degraded significantly. Could you elaborate on the rationale behind your selection of these hyperparameters? Thank you very much.
